src/utilities/request.jl

```diff
@@ -144,7 +144,10 @@ end
 function _http_request(http_backend::HTTPBackend, request::Request, response_stream::IO)
     http_options = merge(http_backend.http_options, request.http_options)
 
-    @repeat 4 try
+    attempts = 4
```

Comment:
More descriptive variable name:
```suggestion
    max_attempts = 4
```